src/stories/Jobs/CanPostJob/endpoint.spec.js

```diff
@@ -11,38 +11,39 @@ describe("Test API Jobs/CanPostJob", () => {
     };
   });
 
-  it("dummy_story_which_will_pass", async () => {
+  it("can_post_job", async () => {
     let respondResult;
     try {
       const app = await httpServer();
 
       const payload = {
-        user_uuid: contextClassRef.user.user_uuid,
-        company_name: "Pogo pvt ltd",
-        title: "Software Engineer",
-        experience: "1 year",
-        location: {
-          country: "India",
-          city: "Hyderabad",
-        },
-        description: "",
-        requirements_attachments: [],
-        top_skills: ["React JS", "Node JS"],
-        employment_type: "private",
-        salary_offer_band: {
-          currency: "INR",
-          range: {
-            from: "500000",
-            to: "600000",
+        recruiter_uuid: "001",
+        title: "Full stack Developer",
+        job_description: {
+          company_name: "google",
+          location: {
+              country: "USA",
+              city: "New York"
           },
-        },
-        about_company: "Open Source IT Projects Private Ltd",
-        company_size: "10-15",
-        industry: "IT - Information Technology",
-        specialization_area: "SAP, Web Development",
-        notice_period_acceptance: "1 Week",
-        note_for_applicants: "",
-        applicants: [],
+          job_types: [
+              "full time",
+              "remote"
+          ],
+          salary_offer_band: {
+              currency: "USD",
+              range: "100000$-200000$"
```

Comment:
I think we might need to discuss this? but lets go with this for now.

Reply:
ok